Gruntfile.js

```diff
@@ -105,10 +106,19 @@ module.exports = function(grunt) {
         },
         command: [
           'rm -rf coverage cov-accept',
-          'env NODE_PATH=. ./node_modules/.bin/istanbul cover --dir cov-accept ./node_modules/.bin/turbo -- --setUp=test/accept/server.js --tearDown=test/accept/server.js test/accept',
+          'env NODE_PATH=. ./node_modules/.bin/istanbul cover --dir cov-accept ./node_modules/.bin/_mocha -- -u exports -R spec ./test/accept/*.js',
           './node_modules/.bin/istanbul report',
           'echo "See html coverage at: `pwd`/coverage/lcov-report/index.html"'
```

Comment:
the coverage of accept tests is at cov-accept/lcov-report/index.html, for unit tests is in cov-unit folder, coverage folder has the coverage for both.
You can change the echo test on line 99 to something like
https://github.com/feedhenry-templates/saml-cloud-app/blob/master/Gruntfile.js#L98
and on line 111 to something like https://github.com/feedhenry-templates/saml-cloud-app/blob/master/Gruntfile.js#L110